modules/tinymce/tools/docs/tinymce.Env.js

```diff
@@ -120,7 +120,7 @@
  * <br>
  * <em>Added in TinyMCE 5.1</em>
  *
- * @method os.isOSX
+ * @method os.isMacOS
  * @return {Boolean} Returns <code>true</code> if the user's operating system is Mac OS X.
```

Comment:
```suggestion
 * @return {Boolean} Returns <code>true</code> if the user's operating system is macOS.
```

Reply:
The description above this also needs `Mac OS X` changed to `macOS` (I can't leave a suggestion on that).

- [ ] Fix macOS name in the API docs